Lib/test/test_unparse.py

```diff
@@ -649,6 +649,21 @@ def test_multiquote_joined_string(self):
         self.check_ast_roundtrip("""f'''""\"''\\'{"\\n\\"'"}''' """)
         self.check_ast_roundtrip("""f'''""\"''\\'{""\"\\n\\"'''""\" '''\\n'''}''' """)
 
+    def test_backslash_in_format_spec(self):
+        self.check_ast_roundtrip("""f"{x:\\}" """)
```

Comment:
Can you add some tests with an uneven number of `\`?

Reply:
Something like this should be valid:

```
>>> class A:
...   def __format__(self, *args):
...      print(args)
...      return "foo"
...
>>> f"{ A():\\\ }"
('\\\\ ',)
'foo'
```

Reply:
you confused me and I checked it.
``` pycon
>>> len("""\\""")
1
```

This example already has 1,2,3 and 4 backslashes. But I added a space at the end like in your example.